src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeInstruct.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```

Comment:
Is this required with HO?  Are they using snake case or camelCase in their responses?

Reply:
yes, HO uses the camelCase so I had to use JsonProperty for some. I followed standards in payments-api. Shall I remove it

Reply:
Removed the naming associated